docs/src/app/components/pages/customization/themes.md

```diff
@@ -18,10 +18,14 @@ import MuiThemeProvider from 'material-ui/lib/MuiThemeProvider';
 import getMuiTheme from 'material-ui/lib/styles/getMuiTheme';
 import AppBar from 'material-ui/lib/app-bar';
 
+// This replaces all text colors with cyan500
```

Comment:
I think we should explain getMuiTheme:

"This calculates the theme from `baseTheme`  and `muiTheme` and then replaces..."?

(If I've understood correctly?)